sdk/ml/azure-ai-ml/azure/ai/ml/entities/_monitoring/signals.py

```diff
@@ -1159,6 +1166,83 @@ def _from_rest_object(cls, obj: RestGenerationSafetyQualityMonitoringSignal) ->
         )
 
 
+@experimental
+class GenerationTokenStatisticsSignal(RestTranslatableMixin):
+    """Generation token statistics signal definition.
+
+    All required parameters must be populated in order to send to Azure.
+
+    :ivar mode: The current notification mode for this signal. Possible values include: "Disabled",
+     "Enabled".
+    :vartype mode: str or ~azure.mgmt.machinelearningservices.models.MonitoringNotificationMode
+    :ivar properties: Property dictionary. Properties can be added, but not removed or altered.
+    :vartype properties: dict[str, str]
+    :ivar signal_type: Required. [Required] Specifies the type of signal to monitor.Constant filled
```

Comment:
Duplicate "required" in a few of the ivars here.

Reply:
correct the documentation in the class file.